10.2/root/etc/my.cnf.d/utf8.cnf

```diff
@@ -0,0 +1,4 @@
+[mysqld]
```

Comment:
@tekante so do you think we should remove this config in 10.2? or just have it as "redundant"?